use-timescale/services/connection-pooling.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Connection pooling
+excerpt: Use a connection pool for your Timescale services
+products: [cloud]
+keywords: [connection pooling, pooler, connections, services]
+cloud_ui:
+    path:
+        - [services, :serviceId]
+---
+
+import EarlyAccess from "versionContent/_partials/_early_access.mdx";
+
+# Connection pooling
+
+You can scale your Timescale connections and improve your database performance
+using connection poolers. Timescale uses `pgBouncer` for connection pooling.
+
+If your database needs a large number of short-lived connections, a connection
+pooler is a great way to improve performance. For example, web, serverless, and
+IoT applications often use an event-based architecture where data is read or
+written from the database for very short amount of time.
+
+<EarlyAccess />
+
+Your application rapidly opens and closes connections while the pooler
+maintains a set of long-running connections to the database. This improves
+performance because the pooler open the connections in advance,  allowing the
+application to open many short-lived connections, while the database opens few,
+long-lived connections.
+
+## Pool types
+
+When you create a connection pooler, there are two pool types to choose from:
+session or transaction. Each pool type uses a different mode to handle
+connections.
+
+Session pools allocate a connection from the pool until they are closed by the
+application, similar to a regular PostgreSQL connection. When the application
+closes the connection, it is sent back to the pool.
+
+Transaction pool connections are allocated only for the duration of the
+transaction, releasing the connection back to the pool when the transaction
+ends. If your application opens and closes connections frequently, choose the
+transaction pool type.
+
+## Connection allocations
+
+A connection pooler manages connections to both the database itself, and the
+client application.
+
+The number of database connections allocated in each pool is managed by the
+`max_connections` parameter set in the service. The pooler keeps allocating
+connections until 60% of the total `max_connections`. 45% of the
+`max_connections` parameter is used for the session pool, and the remaining 15%
+is available for the transaction pool.
+
+The connection pooler can then allocate connections for the client application.
+Up to 45% of `max_connections` is available for the session pool, and up to 100%
+of `max_connections` is available for the transaction pool.
+
+For example, if `max_connections` is set to 500, the maximum number of database
```

Comment:
We should add that the customer can always make use of all requested `max_connections` by connecting directly, the limits discussed in this paragraph apply only to the pooler connections.

Reply:
I've added a sentence, but I'm not 100% certain it makes sense. What does "connect directly" mean, in practice?

Reply:
Your addition make perfect sense in this context, thank you!